src/styles/icons/ab_icons.css

```diff
@@ -315,9 +315,6 @@
 .icon-ab-sidebar-campaigns-2:before {
   content: "\e95f";
 }
-.icon-ab-config-promoter:before {
```

Comment:
please recheck accelicons.svg, arrange-top and arrange-bottom content should change

Reply:
no, wahyu just remove the config-promoter, no other changes

Reply:
if we do it correctly, both in svg and ab_icons.css, arrange-top content should be \e960 and arrage-bottom should be \e961

Reply:
if we remove an icon, we should reorder the unicode of other icons after it

Reply:
i see, but wahyu didn't handle it so meticulously

Reply:
![image](https://user-images.githubusercontent.com/93185957/209276839-7a14fc3b-861d-4191-9086-111f57672528.png)
in accelicons.svg, the unicode should change to `&#xe960` and `&#xe961`. then we should also update the content of those two icons in ab_icons.css. this is to maintain the icons' order and there should be no skipped order (currently in this case \e960 is skipped)

Reply:
you can refer to a confluence doc with keyword `How to add new icons to accelicons` when you wanna make an adjustment to accelicons

Reply:
> ![image](https://user-images.githubusercontent.com/93185957/209276839-7a14fc3b-861d-4191-9086-111f57672528.png) in accelicons.svg, the unicode should change to `&#xe960` and `&#xe961`. then we should also update the content of those two icons in ab_icons.css. this is to maintain the icons' order and there should be no skipped order (currently in this case \e960 is skipped)

This file is no longer used, and I understand your request for order, I will let wahyu adjusts it

Reply:
we adjust it, not wahyu

Reply:
> we adjust it. wahyu doesn't need to do anything

But what is really used is this file accelicons.woff, I do not have iconmoon account to change it, maybe you can help

Reply:
okay, can we not bump the version? as this change won't affect AP anyway